packages/react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -2337,11 +2337,11 @@ class ComboBoxInternal extends React.Component<IComboBoxInternalProps, IComboBox
   /**
    * Get the aria autocomplete value for the combo box
    * @returns 'inline' if auto-complete automatically dynamic, 'both' if we have a list of possible values to pick from
-   * and can dynamically populate input, and 'none' if auto-complete is not enabled as we can't give user inputs.
+   * and can dynamically populate input, and 'list' if auto-complete is not enabled as selection is the only option.
```

Comment:
Could you actually drop a comment in here about the Edge behavior so we know why we chose `list` over `none` in the future?

Reply:
Added